caos.ansible_roles/roles/package_install/tasks/package-RedHat.yaml

```diff
@@ -12,6 +12,7 @@
     name: "{{ versioned_package }}"
     update_cache: yes
     state: "{{ state }}"
+    update_only: yes #for some reason w/o this package is not being updated in al2
```

Comment:
This was lost when moving from Infra Agent https://github.com/newrelic/infrastructure-agent/blob/a125c5ec08d51e375d6432b8c44e9d5c78f4af5c/test/packaging/ansible/roles/agent-upgrade/tasks/upgrade-RedHat.yaml